test/scripts/install-kind.sh

```diff
@@ -2,23 +2,30 @@
 
 set -ex
 
-export GO111MODULE="on"
+# Install latest stable kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
 
-pushd $GOPATH/src/k8s.io/kubernetes/
-if [[ ! -f /usr/local/bin/kubectl ]]; then
-  sudo ln ./_output/local/go/bin/kubectl /usr/local/bin/kubectl
-fi
-if [[ ! -f /usr/local/bin/e2e.test ]]; then
-  sudo ln ./_output/local/go/bin/e2e.test /usr/local/bin/e2e.test
-fi
-popd
+# Install latest e2e test binary
+# The e2e binaries are built from https://github.com/aojea/kubernetes-e2e-binaries
+# Current e2e binary versions is latest 1.19 tag v1.19.4
+curl -LO https://github.com/aojea/kubernetes-e2e-binaries/releases/download/d360454c/e2e.test
```

Comment:
would prefer if we just make an ovn-org repo to do this instead of it being in @aojea repo...thoughts @dcbw @danwinship @girishmg ?

Reply:
same here, let me see if I can find the upstream binaries, I know that they publish them somewhere for the CI

Reply:
no worries, I´ve managed to get the URL to the upstream binaries that is a much better solution and are published for these situations, so projects in the ecosystem don´t have to do the building that is expensive in terms of resources
PTAL to the new commit